include/vkfw/vkfw.hpp

```diff
@@ -83,6 +83,21 @@
   #include <vulkan/vulkan.hpp>
 #endif
 
+#if defined(VULKAN_HPP_NO_SMART_HANDLE) && !defined(VKFW_NO_SMART_HANDLE)                          \
+  && !defined(VKFW_NO_INCLUDE_VULKAN_HPP)
+  #if defined(_MSC_VER)
+    #pragma message(                                                                               \
+      "warning: VKFW_NO_SMART_HANDLE will be defined as vkfw requires Vulkan-HPP handles when "    \
+      "VKFW_NO_INCLUDE_VULKAN_HPP isn't defined. You can silence this warning by defining "        \
+      "VKFW_NO_SMART_HANDLE or VKFW_NO_INCLUDE_VULKAN_HPP")
+  #else
+    #warning "VKFW_NO_SMART_HANDLE will be defined as vkfw requires Vulkan-HPP handles when \
+VKFW_NO_INCLUDE_VULKAN_HPP isn't defined. You can silence this warning by defining \
+VKFW_NO_SMART_HANDLE or VKFW_NO_INCLUDE_VULKAN_HPP"
+  #endif
```

Comment:
Ah, right, I totally forgot about this. Good catch!

But looking into it, both `gcc` and `clang` seem to support `#pragma message` so let's just get rid of `#warning` to avoid having to spell the error twice.